src/patternfly/components/ContextSelector/context-selector.scss

```diff
@@ -118,6 +128,38 @@
     }
   }
 
+  &.pf-m-plain {
+    --pf-c-context-selector__toggle--PaddingRight: var(--pf-c-context-selector__toggle--m-plain--PaddingRight);
+    --pf-c-context-selector__toggle--PaddingLeft: var(--pf-c-context-selector__toggle--m-plain--PaddingLeft);
+
+    &:not(.pf-m-text) {
+      justify-content: center;
+      width: auto;
+      color: var(--pf-c-context-selector__toggle--m-plain--Color);
+    }
+
+    &::before {
+      border: 0;
+    }
+
+    .pf-c-context-selector__toggle-button-icon {
+      line-height: var(--pf-c-context-selector__toggle--LineHeight);
```

Comment:
Doesn't look like you need this - the var `--pf-c-context-selector__toggle--LineHeight` doesn't exist either. 

Doesn't look like we need it in the options menu, either. Would you mind creating an issue to remove this line from the options menu component and add a "breaking change" label (since we're removing a variable). https://github.com/patternfly/patternfly/blob/005fa69c06705950c767354cd9d667d3d90a75bc/src/patternfly/components/OptionsMenu/options-menu.scss#L174